### PR TITLE
BaSP-TG-149: Super admin view

### PR DESCRIPTION
**Pull request by fede230298**
Create view from users with Super Admin Role.
Profile Page + Edit form
CRUD Admins

### CODE REVIEW

File: src/Components/Layout/SuperAdminLayout/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
+import React, { lazy, Suspense } from 'react';
+import { Switch, Route, Redirect, useLocation } from 'react-router-dom';
+import createTitle from 'Helpers/create-title.js';
+import Header from 'Components/Shared/Header/index';
+import Sidebar from 'Components/Shared/Sidebar';
+
+const Admins = lazy(() => import('Components/Pages/SuperAdmin/Admins/AdminList/index'));
+const AdminForm = lazy(() => import('Components/Pages/SuperAdmin/Admins/AdminForm/index'));
+const MyProfile = lazy(() => import('Components/Pages/SuperAdmin/MyProfile/'));
```
Comment (ProDIGGY-vinilos) [inline src/Components/Layout/SuperAdminLayout/index.js:10]: Maybe you could add the index name to these two to have the 4 lazy identical

File: src/Components/Pages/SuperAdmin/Admins/AdminForm/index.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+          error={errors.password?.message}
+        />
+      </div>
+      <div>
+        <MessageModal
+          type={typeModal}
+          isOpen={showMessageModal}
+          message={textMessageModal}
+          handleClose={closeMessageModal}
+          goBack={'/admin/admins'}
```
Comment (ProDIGGY-vinilos) [inline src/Components/Pages/SuperAdmin/Admins/AdminForm/index.js:159]: This redirect is ok?

File: src/Components/Pages/SuperAdmin/MyProfile/MyProfileForm/index.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  };
+
+  return (
+    <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
+      <div className={styles.formHeader}>
+        <h3 id="fromHeader">Title</h3>
+        <Button href="/super-admin/profile" style="roundedSecondary" disabled={false} text="X" />
+      </div>
+      <div className={styles.fromInput}>
+        <InputField
```
Comment (ProDIGGY-vinilos) [inline src/Components/Pages/SuperAdmin/MyProfile/MyProfileForm/index.js:123]: From or form input?

File: src/Components/Pages/SuperAdmin/Admins/AdminForm/index.js (hunk 2 above)
Comment (fede230298) [reply src/Components/Pages/SuperAdmin/Admins/AdminForm/index.js:159]: Done

File: src/Components/Layout/SuperAdminLayout/index.js (hunk 1 above)
Comment (fede230298) [reply src/Components/Layout/SuperAdminLayout/index.js:10]: Done

File: src/Components/Pages/SuperAdmin/MyProfile/MyProfileForm/index.js (hunk 3 above)
Comment (fede230298) [reply src/Components/Pages/SuperAdmin/MyProfile/MyProfileForm/index.js:123]: form input, form is used in form tag. Copy from admin styles created originally.

Comment (fede230298) [reply src/Components/Pages/SuperAdmin/MyProfile/MyProfileForm/index.js:123]: Done

**Review by JuliMonte5 — APPROVED** [review]

**Pull request merged**